cloud/pkg/edgecontroller/manager/node.go

```diff
@@ -1,30 +0,0 @@
-package manager
-
-import (
-	"k8s.io/apimachinery/pkg/watch"
-	"k8s.io/client-go/tools/cache"
-)
-
-const (
-	NodeRoleKey   = "node-role.kubernetes.io/edge"
-	NodeRoleValue = ""
-)
-
-// NodesManager manage all events of nodes by SharedInformer
-type NodesManager struct {
-	events chan watch.Event
-}
-
-// Events return the channel save events from watch nodes change
-func (nm *NodesManager) Events() chan watch.Event {
-	return nm.events
-}
-
-// NewNodesManager create NodesManager by kube clientset and namespace
-func NewNodesManager(si cache.SharedIndexInformer) (*NodesManager, error) {
-	events := make(chan watch.Event)
```

Comment:
Is len of this event buffer equal to 0 set by design? Or just a mistake. If the former, we should take it  consideration

Reply:
I think it is just a mistake, for node resource we do not have some special logic.